app/controllers/saml_sessions_controller.rb

```diff
@@ -17,11 +17,8 @@ def destroy
     session["signed_out"] = true
     session["feedback_return_path"] = destroy_provider_session_path
     sign_out current_provider
-    if IdPSettingsAdapter.mock_saml?
-      redirect_to providers_root_url
-    else
-      redirect_to new_feedback_path
-    end
+
+    redirect_to new_feedback_path, notice: t(".success")
```

Comment:
Design: the flash notice message uses the govuk warning component, may be better to use another component, for example the notification component. Check component usage or speak with design profession.

Reply:
Discussed this with design, and they have confirmed we can use the notification component in place of the warning text for flash notice messages.